internal/testplanet/satellite.go

```diff
@@ -154,10 +154,11 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 				ReliabilityCacheStaleness: 5 * time.Minute,
 			},
 			Repairer: repairer.Config{
-				MaxRepair:    10,
-				Interval:     time.Hour,
-				Timeout:      1 * time.Minute, // Repairs can take up to 10 seconds. Leaving room for outliers
-				MaxBufferMem: 4 * memory.MiB,
+				MaxRepair:                     10,
+				Interval:                      time.Hour,
+				Timeout:                       1 * time.Minute, // Repairs can take up to 10 seconds. Leaving room for outliers
+				MaxBufferMem:                  4 * memory.MiB,
+				MaxExcessRateOptimalThreshold: 0.05,
```

Comment:
Unsure if it's necessary, but we might need to let others know that there is this new config value in the config-changes channel

Reply:
It was needed, it's something that @mobyvb reported, otherwise it's 0 and I believe that's better that our tests use the default value rather than using 0 and have to configure the specific test that it test the behavior that this configuration value involves.

The changes in the configuration channel were reported and approved; that was the green light to me to update the configuration lock file.

Thanks for raising the point.